app/src/ui/diff/diff-helpers.tsx

```diff
@@ -393,3 +393,63 @@ export function getNumberOfDigits(val: number): number {
  * GitHub.com's behavior.
  **/
 export const MaxIntraLineDiffStringLength = 1024
+
+/**
+ * Used to obtain classes applied to style the row as first or last of a group
+ * of added or deleted rows in the side-by-side diff.
+ **/
+export function getFirstAndLastClassesSideBySide(
+  row: SimplifiedDiffRow,
+  previousRow: SimplifiedDiffRow | undefined,
+  nextRow: SimplifiedDiffRow | undefined,
+  addedOrDeleted: DiffRowType
```

Comment:
I wonder if we could make this even more clear by explicitly stating the types we allow here

```suggestion
  addedOrDeleted: DiffRowType.Added | DiffRowType.Deleted
```